openfpga_flow/scripts/run_fpga_flow.py

```diff
@@ -489,6 +489,21 @@ def run_yosys_with_abc():
         "LUT_SIZE": lut_size,
         "OUTPUT_BLIF": args.top_module+"_yosys_out.blif",
     }
+
+    for indx in range(0, len(OpenFPGAArgs), 2):
+        tmpVar = OpenFPGAArgs[indx][2:].upper()
+        ys_params[tmpVar] = OpenFPGAArgs[indx+1]
+    
+    if 'YOSYS_FAMILY' not in ys_params.keys():
+        # define default family as 'qlf_k4n8'
+        ys_params['YOSYS_FAMILY'] = "qlf_k4n8"
```

Comment:
I do not see a strong motivation to hard code an default parameter here. Users may not be aware of any default values and misuse them. I would suggest always defining these parameters explicit clearly in task configuration files.